src/app/lib/config/services/tigrinya.js

```diff
@@ -43,11 +43,11 @@ const service = {
       brandLogoColour: `${C_WHITE}`,
     },
     translations: {
-      seeAll: 'See all',
+      seeAll: 'ኩሉ ረኣይ',
       home: 'ዜና',
-      currentPage: 'Current page',
+      currentPage: 'ህሉው ገጽ',
       skipLinkText: 'እቲ ትሕዝቶ ዝለል',
```

Comment:
This has changed in the spreadsheet.

Reply:
mmm I see the same one `ህሉው ገጽ`

Reply:
Sorry, this is for the skipLinkText: `ናብቲ ትሕዝቶ ቀጽል`

Reply:
Done! They are faster than us :P